pkg/ccl/streamingccl/streamingest/stream_ingestion_planning.go

```diff
@@ -273,6 +278,7 @@ func ingestionPlanHook(
 			destinationTenantID,
 			retentionTTLSeconds,
 			options.resumeTimestamp,
+			revertFirst,
```

Comment:
nit: could we hardcode this to false?

Reply:
is this a question of fact or request/suggestion:
as far as fact, yes, we can -- it is set to false in both sides of the conditional above. I had `false /* blah */` but switched to this since it let me comment why it is false for each case in that case.

That said, I think we're deleting one of the two cases in the next week so maybe this is moot.

Reply:
cool, fine to leave as is.